src/embed.tsx

```
@@ -9,34 +9,58 @@ import { TooltipProvider } from "@/components/ui/tooltip";
 
 // Wrapper component to control welcome modal opening
 const EmbeddedApp = ({ variants, onUpdate, onApply, onClose }) => {
   // Effect to control when the welcome modal appears in embed context
+  const safeCookieIncludes = (needle: string) => {
+    try {
+      return typeof document !== 'undefined' && document.cookie.includes(needle);
+    } catch {
+      return false;
+      try {
+        localStorage.setItem('foodcube-welcome-modal-shown', 'true');
+      } catch {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)